doc/modules/clustering.rst

```diff
@@ -743,6 +750,77 @@ each class.
 
  * :ref:`sphx_glr_auto_examples_cluster_plot_agglomerative_clustering_metrics.py`
 
+Bisecting K-Means
+-----------------
+
+.. _bisect_k_means:
+
+The :class:`BisectKMeans` is an iterative variant of :class:`KMeans`, using
+divisive hierarchical clustering.
+Instead of creating all centroids at once - It obtains desired number of centroids
+by picking one cluster at each iteration and splitting it into two new clusters,
+until target number of clusters is reached.
+
+With that approach, it is possible to produce either an un-nested (flat)
+clustering or a hierarchical clustering.
+That approach is also preferable to agglomerative clustering
+if the number of clusters is small, compared to the number of data points.
+
+Cluster to split is picked depending on parameter ``bisect_strategy``:
+- Cluster with largest amount of data points (with ``largest_cluster``)
```

Comment:
This does not render as a list. You might need to add two whitespaces before the dash.